src/api/events/window.ts

```diff
@@ -211,8 +192,6 @@ export interface PropagatedWindowEventMapping<Topic = string, Type = string> ext
     'window-performance-report': WindowPerformanceReport<Topic, Type>;
     'window-preload-scripts-state-changed': WindowPreloadScriptsStateChangeEvent<Topic, Type>;
     'window-preload-scripts-state-changing': WindowPreloadScriptsStateChangedEvent<Topic, Type>;
-    'window-resource-load-failed': WindowResourceLoadFailedEvent<Topic, Type>;
```

Comment:
These should be left in unless we start propagating webcontents events differently

Reply:
This is from your `hold` commit: 
https://github.com/HadoukenIO/js-adapter/pull/343/commits/9a7d9abc5b2f3d66019e0861c5eb0509ce1287a3#diff-ba47ceacf9a1245b0bb67531547c3d20L204

Reply:
I messed up then

Reply:
We might move it into a `propagatedWebcontentsEventMapping` when we add event propagation but for now this should stay.

Reply:
Re-added